packages/react/navigation-menu/src/NavigationMenu.tsx

```diff
@@ -197,6 +210,8 @@ const NavigationMenuProvider: React.FC<NavigationMenuProviderProps> = (
     defaultValue,
     dir,
     orientation,
+    delayDuration = DEFAULT_DELAY_DURATION,
+    skipDelayDuration = 300,
```

Comment:
I think I'd prefer seeing the defaults being applied at the props level in the components, rather than on the provider.
I know this means multiple places but I think it's better that way so we can always assume a value internally anywhere.